### PR TITLE
Login Screen Updates 

### PR DESCRIPTION
**Pull request by tarzzz**
- [x] Tested on on-prem

### CODE REVIEW

**Comment by tarzzz** [comment]
@n-riesco or @chriddyp Please review..

Screenshot (On Prem):

screen shot 2017-12-08 at 5 32 06 pm


cc: @scjody ..

**Comment by tarzzz** [comment]
One small concern:

The title text is directly fetched from `package.json`'s `version` value, so it currently it reflects `beta` which IMHO is not right for an on-prem release:

screen shot 2017-12-08 at 6 05 24 pm

We need to either:
1. Update the package.json to *not* be beta for releases..
2. Use a different value to be appended for header
3. Remove the `version` from the title -- I am in favor of this as this requires lesser maintenance. The title will just say 'Falcon SQL Client'.

Please let me know your thoughts on this..

**Comment by n-riesco** [comment]
@tarzzz

> The title text is directly fetched from package.json's version value, so it currently it reflects beta which IMHO is not right for an on-prem release

An option that requires no code changes is to update the version in `package.json` once the onprem branch is created.

File: app/components/Login.react.js [hunk 1]
```diff
@@ -48,11 +50,10 @@ class Login extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            domain: '',
+            domain: (isOnPrem() ? plotlyUrl() : 'https://plot.ly'),
             statusMessage: '',
             serverType: CLOUD,
```
Comment (n-riesco) [inline app/components/Login.react.js]: Does `serverType` need to be set to `ONPREM` if `isOnPrem()` is `true`?

**Comment by scjody** [comment]
This looks great! I'll let someone more familiar with the codebase do the review.

Regarding the title text, please fix it in this PR or else open a new issue in streambed in the 2.3.0 milestone. I think @n-riesco's suggestion there is a good one, or removing the version. (I've never seen any other software product that included the version in the title of the page so I think that idea is worth considering.)

**Comment by n-riesco** [comment]
@tarzzz It looks the linter doesn't catch indentation issues inside JSX (by chance I noticed an indentation mismatch [here](https://github.com/plotly/falcon-sql-client/blob/2a3d8abda9bcfc3401bc2db0595c0094d4fd1642/app/components/Login.react.js#L297)).

When I tested Login into Plotly Cloud:
- it worked using the desktop
- but it failed using the dockerised app (issue #260)

I tried to test Plotly OnPrem but it looks like the OnPrem domain I was using is obsolete.

Is #260 something onprem would ask us to fix?

**Comment by tarzzz** [comment]
> It looks like the linter doesn't catch indentation issues inside JSX 

👍 Will update it .. 

> I think @n-riesco's suggestion there is a good one, or removing the version

I am going to remove the version.. It looks nicer imo..

> Is #260 something onprem would ask us to fix?

Let's take the conversation related to #260 on the issue itself..

**Comment by n-riesco** [comment]
> > I think @n-riesco's suggestion there is a good one, or removing the version

> I am going to remove the version.. It looks nicer imo..

If you want to go that way, you need to find an alternative place to show the version, so that I can ask users what version they are using.

**Comment by tarzzz** [comment]
@n-riesco Addressed the indentation issues and bumped version to `2.3.4` . Let's leave the title as it is for now.

**Comment by n-riesco** [comment]
:dancer:

**Pull request merged**